src/Model/LoginSession.php

```diff
@@ -174,12 +182,12 @@ public function handlePermission(string $funcName, $member): bool
         }
 
         // Members can manage their own sessions
-        if ($this->ID == $member->ID) {
+        if ($this->MemberID === $member->ID) {
```

Comment:
This will cause a merge conflict with #59 who does the same thing. We'll just have to deal with it once we get to it.